doc/source/admin/conda_faq.rst

```diff
@@ -234,11 +265,11 @@ and assist with all development. You will also see a few of us at this
 project to get you started :)
 
 Don't be scared! Conda recipes are really simple to write. Conda also
-offers you, so called \`skeleton\` generators, generating recipes from
-pypi, cran, cpan for you (mostly) automatically.
+offers so called \`skeleton\` generators that generate recipes from
+pypi, cran, or cpan for you (mostly) automatically.
 
 
-9. Is there a way to convert traditional Tool Shed package recipes that are not yet in a Conda channel?
+11. Is there a way to convert traditional Tool Shed package recipes that are not yet in a Conda channel?
```

Comment:
This is out of order now right? 10 seems to come after this - or is that an artifact of the github diff.